client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.jsx

```diff
@@ -69,6 +76,31 @@ const PrePurchaseNotices = () => {
 		return products.filter( ( p ) => ! p.expired );
 	} );
 
+	const akismetProductSlugThatOverlapsCartProduct = useMemo( () => {
+		const productSlugInCart = cartItemSlugs.find( ( productSlug ) =>
+			isAkismetProduct( { productSlug } )
+		);
+		const akismetPurchases = userActivePurchases.filter( isAkismetProduct );
+
+		// Continue only if the cart includes Akismet product
+		if ( ! productSlugInCart ) {
+			return null;
+		}
+
+		const lowerTierProducts = akismetPurchases.filter(
+			( { productSlug } ) =>
+				AKISMET_PRODUCTS_LIST.indexOf( productSlug ) -
+					AKISMET_PRODUCTS_LIST.indexOf( productSlugInCart ) <
+				0
```

Comment:
Could this be simplified to 
`AKISMET_PRODUCTS_LIST.indexOf( productSlugInCart ) > AKISMET_PRODUCTS_LIST.indexOf( productSlug )`

I think this works if we assume the product list array will always be in order of progressive tiers. Would there ever be a case where that is not true?